### PR TITLE
skip verify-flaws for preview and prerelease

### PR DESCRIPTION
**Pull request by ashwindasr**
Skip `--verify-flaws` flag for elliott verify-attached-bugs for preview and prerelease

### CODE REVIEW

File: pyartcd/pyartcd/pipelines/promote.py [hunk 1]
```diff
@@ -881,7 +886,8 @@ def verify_advisory_status(self, advisory_info: Dict):
         if advisory_info["status"] not in {"QE", "REL_PREP", "PUSH_READY", "IN_PUSH", "SHIPPED_LIVE"}:
             raise VerificationError(f"Advisory {advisory_info['id']} should not be in {advisory_info['status']} state.")
 
-    async def verify_attached_bugs(self, advisories: Iterable[int], no_verify_blocking_bugs: bool):
+    async def verify_attached_bugs(self, advisories: Iterable[int], no_verify_blocking_bugs: bool,
+                                   skip_verify_flaws: bool):
```
Comment (thegreyd) [inline pyartcd/pyartcd/pipelines/promote.py:890]:
```suggestion
                                  verify_flaws: bool = True):
```

Comment (thegreyd) [reply pyartcd/pyartcd/pipelines/promote.py:890]: Would recommend having a verify_flaws param that is set to true by default and then we set it to False when we deem it necessary

Comment (ashwindasr) [reply pyartcd/pyartcd/pipelines/promote.py]: done! repushed

**Pull request merged**